### PR TITLE
README minor corrections

### PR DESCRIPTION
**Pull request by mauro-moreno**
README $lab / $labratory changed to $laboratory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,21 +27,21 @@ A PHP experiment library inspired by Github's own [Scientist](https://github.com
 
 Require the latest version of Scientist using [Composer](https://getcomposer.org/).
 
-    composer require daylerees/scientist
+    composer require --dev daylerees/scientist
```
Comment (daylerees) [inline README.md]: Scientist should be able to run in production. So we don't need this flag. :)

**Comment by daylerees** [comment]
Need to remove the `--dev` but the rest looks good. Thanks. :)

**Comment by mauro-moreno** [comment]
Nice, I was thinking about creating the same project so I'm going to try to help in whatever I can. Kudos

**Comment by daylerees** [comment]
Nice :) thank you! I'll have a contributors guide soon too. There will be plenty of ideas on there!

**Pull request merged**